main.go

```diff
@@ -390,9 +390,17 @@ func initDaemon(config *menderConfig, dev installer.DualRootfsDevice, env instal
 	mp.dualRootfsDevice = dev
 
 	controller, err := NewMender(config, *mp)
-	if controller == nil {
-		mp.store.Close()
-		return nil, errors.Wrap(err, "error initializing mender controller")
+	if err != nil {
+		// Ignore server certificate error  (See: MEN-2378)
+		if _, ok := errors.Cause(err).(*client.ClientServerCertificateError); ok {
+			errstr := "IGNORING ERROR: The client server-certificate can not be loaded error: (%v). The client will " +
+				"continue running, but will not be able to communicate with the server. If this is not your intention " +
+				"please add a valid server certificate"
+			log.Warningf(errstr, err)
```

Comment:
Please use `%s` and `err.Error()` instead. I've fixed so many logging bugs that were not caught by the static code analyzers because `%v` accepts any argument.

Reply:
Haha, did not know this was an issue. Fixing.